src/main/java/org/terracotta/build/plugins/CustomHostPluginExtension.java

```diff
@@ -0,0 +1,7 @@
+package org.terracotta.build.plugins;
+
+import org.gradle.api.provider.Property;
+
+public interface CustomHostPluginExtension {
+  Property<String> getCustomHosts();
```

Comment:
Wouldn't this be better represented as something with more structure than a bare string? At a minimum a list of strings... or maybe a map of strings to a list of strings... or maybe something even richer?